java-frontend/src/main/java/org/sonar/java/se/checks/XxeProcessingCheck.java

```diff
@@ -351,6 +363,23 @@ public ProgramState checkPostStatement(CheckerContext context, Tree syntaxNode)
     return visitor.programState;
   }
 
+  private static ProgramState addNamedConstraint(@Nullable ExpressionTree expressionTree, ProgramState state) {
+    if (expressionTree != null) {
+      SymbolicValue sv = state.peekValue();
+      if (sv != null) {
+        Optional<String> value = expressionTree.asConstant(String.class);
+        if (value.isPresent()) {
+          for (XxeProperty property : PROPERTIES_TO_CHECK) {
+            if (property.isNamed(value.get())) {
```

Comment:
what about `property.propertyName().equals(value.get())` and drop `isNamed`? or maybe we don't need anymore `propertyName()`